pandas/tests/plotting/test_frame.py

```diff
@@ -3256,6 +3257,117 @@ def test_subplots_sharex_false(self):
         tm.assert_numpy_array_equal(axs[0].get_xticks(), expected_ax1)
         tm.assert_numpy_array_equal(axs[1].get_xticks(), expected_ax2)
 
+    @pytest.mark.parametrize("by", ["C", ["C", "D"]])
+    @pytest.mark.parametrize("column", ["A", ["A", "B"]])
+    def test_hist_plot_by_argument(self, by, column):
+        # GH 15079
+        df = DataFrame(np.random.randn(30, 2), columns=["A", "B"])
```

Comment:
Rather than repeat this frame construction in every test you can just make a fixture for it in the global namespace

Reply:
yeah, nice idea! changed!